osdep/terminal-unix.c

```diff
@@ -407,10 +409,16 @@ void terminal_setup_getch(struct input_ctx *ictx)
     if (mp_make_wakeup_pipe(death_pipe) < 0)
         return;
 
+    tty_in = tty_out = open("/dev/tty", O_RDWR);
```

Comment:
Even though in 99% of all cases the application will finish by closing itself, I would guess it's good manners to close the file descriptor at the end around `terminal_uninit`.

Additionally, the POSIX [CLOEXEC](http://pubs.opengroup.org/onlinepubs/9699919799/functions/open.html) flag could be utilized when `open()`'ing the file.